src/main/java/levi9/VehicleService/service/impl/VehicleServiceImpl.java

```diff
@@ -0,0 +1,31 @@
+package levi9.VehicleService.service.impl;
+
+import levi9.VehicleService.dto.VehicleDto;
+import levi9.VehicleService.model.Vehicle;
+import levi9.VehicleService.repository.VehicleRepository;
+import levi9.VehicleService.service.RentService;
+import levi9.VehicleService.service.VehicleService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.modelmapper.ModelMapper;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class VehicleServiceImpl implements VehicleService {
+    @Autowired
+    private VehicleRepository vehicleRepository;
+    @Autowired
```

Comment:
With `@AllArgsConstructor` on class above you don't need to put `@Autowired` on every injected field.